packages/mesos/extra/dcos-drain-mesos-agent-public.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Drain Mesos Agent: Stop Mesos tasks on shutdown/reboot
+Before=shutdown.target reboot.target halt.target
+After=dcos-mesos-slave-public.service
```

Comment:
nit: Since the only difference between the two units is the name of the Mesos service, we could turn this file into a template with the unit name parameterized. This file could reference the env var, and the build script could set its value and then render the actual unit file with `envsubst`. The Mesos build file is already doing something similar with other units.

Reply:
Good suggestion. I was just lazy followed what I already saw with dcos-mesos-slave-public.service and dcos-mesos-slave.service. I will template it and update the PR